pl_bolts/utils/semi_supervised.py

```diff
@@ -51,19 +53,19 @@ def balance_classes(X: np.ndarray, Y: list, batch_size: int):
     nb_batches = math.ceil(len(Y) / batch_size)
 
     # sort by classes
-    final_batches_x = [[] for i in range(nb_batches)]
-    final_batches_y = [[] for i in range(nb_batches)]
+    final_batches_x: List[Any] = [[] for i in range(nb_batches)]
+    final_batches_y: List[Any] = [[] for i in range(nb_batches)]
```

Comment:
In my opinion, we shouldn't use `Any` where possible, so can we rewrite it as follows?
```suggestion
    final_batches_x: List[List[int]] = [[] for i in range(nb_batches)]
    final_batches_y: List[List[int]] = [[] for i in range(nb_batches)]
```

Reply:
Yes, I was not sure regarding the type, so keep it as `Any`, will change that now.

Reply:
Sorry, what I've just mentioned is completely false statement.  Still, it would be nice to have a more specific type, but we can keep it as is since it seems not obvious...

Reply:
Okay, sure.